src/components/HURUmap/LocationHeader/index.js

```diff
@@ -1,10 +1,11 @@
 import { Typography, Button } from "@material-ui/core";
-import Icon from "next/image";
 import PropTypes from "prop-types";
 import React from "react";
 
 import useStyles from "./useStyles";
 
+import Icon from "@/pesayetu/components/Image";
```

Comment:
Some places you're importing this as `Icon`, some as `Image`. Lets be consistent. I suggest `Image`.. over to you.